share/spack/bootstrap/github-actions/clingo.json

```diff
@@ -0,0 +1,257 @@
+      "spec": "clingo-bootstrap%gcc platform=linux target=ppc64le"
+    },
+    {
+      "binaries": [
+        [
+          "clingo-bootstrap",
+          "i2uoawqsmmkr2uwb42oxfsnt4djzejrj",
+          "6511837f87e50c40a7f4aab2ec7454e5200594821e4d1fc4a441d3be647b9acb"
+        ]
+      ],
+      "python": "python@3.9",
+      "spec": "clingo-bootstrap%gcc platform=linux target=ppc64le"
+    },
+    {
+      "binaries": [
+        [
+          "clingo-bootstrap",
+          "p3bqplnwqo66lvvolmtaezck77jafrc4",
+          "436f9483e4028c12c32ba9f4e7e91e944bf9819ef487dfe4e42ddd1d487c93ee"
+        ]
+      ],
+      "python": "python@2.6",
+      "spec": "clingo-bootstrap%gcc platform=linux target=x86_64"
+    },
+    {
+      "binaries": [
+        [
+          "clingo-bootstrap",
+          "rpa6yyujr7ilnfcowq2pbmkmh7uzrijp",
+          "3be0f4ccd412d45506f9f305ef7f6621cd246fbde97aed081595d01dafe3c397"
+        ]
+      ],
+      "python": "python@2.7+ucs4",
+      "spec": "clingo-bootstrap%gcc platform=linux target=x86_64"
+    },
+    {
+      "binaries": [
+        [
+          "clingo-bootstrap",
+          "y32mbn7swer3yxvgf6tmkgekpo23uo5z",
+          "f6e0716bd97f2df123abcd96ec8884c525a9fd10b81e0062784e7b0d2df3f622"
+        ]
+      ],
+      "python": "python@2.7~ucs4",
+      "spec": "clingo-bootstrap%gcc platform=linux target=x86_64"
+    },
+    {
+      "binaries": [
+        [
+          "clingo-bootstrap",
+          "ohtmyp5c74xt75csx4llbjs5anobryp6",
+          "7d613ddbca1640d761311fb00403c0cb65e279534c44a2129b8d9610f6146e78"
+        ]
+      ],
+      "python": "python@3.5",
+      "spec": "clingo-bootstrap%gcc platform=linux target=x86_64"
+    },
+    {
+      "binaries": [
+        [
+          "clingo-bootstrap",
+          "vcipwnf57slgoo7busvvkzjkk7vydeb5",
+          "db5222760045f20ad1e5c194179d31273b8e4bfa6ade38e15cd3182d685cc05b"
+        ]
+      ],
+      "python": "python@3.6",
+      "spec": "clingo-bootstrap%gcc platform=linux target=x86_64"
+    },
+    {
+      "binaries": [
+        [
+          "clingo-bootstrap",
+          "ffoyoewfd6pdwbjniodfkqusyvkrbhyi",
+          "14cea5f6cfd86bcb8de38ad8c1a5e44cc22955de2e7c78b825b617dccd107dbe"
+        ]
+      ],
+      "python": "python@3.7",
+      "spec": "clingo-bootstrap%gcc platform=linux target=x86_64"
+    },
+    {
+      "binaries": [
+        [
+          "clingo-bootstrap",
+          "kt74l7kjzrlp3cgtj2576o33mhsrgyrw",
+          "e71de4beb68bb3e58bd2dcb98dc3be3a375c82781b6f7cb01bc5d552c2240bd2"
+        ]
+      ],
+      "python": "python@3.8",
+      "spec": "clingo-bootstrap%gcc platform=linux target=x86_64"
+    },
+    {
+      "binaries": [
+        [
+          "clingo-bootstrap",
+          "hmnv6gk5wha64k6r3s7hid35mzvhkuot",
+          "b08ff59357fa184ce39b8cc0a17aaf7f0a925a449ab389e1afa4eab6ae026f2e"
+        ]
+      ],
+      "python": "python@3.9",
+      "spec": "clingo-bootstrap%gcc platform=linux target=x86_64"
+    }
+  ]
+}
```

Comment:
Probably can be a follow-on PR, but we need a way for users to automatically generate this json file from their own binary cache.

Reply:
I agree. I was thinking of adding it in a subsequent PR where we add to `spack bootstrap` the capability to register custom buildcaches for bootstrapping.